mongodb-datastore/db/mongodb_event_repo.go

```diff
@@ -155,7 +155,7 @@ func (mr *MongoDBEventRepo) DropProjectCollections(event keptnapi.KeptnContextEx
 	err = rootEventsCollection.Drop(ctx)
 	if err != nil {
 		err := fmt.Errorf(dropCollectionErrorMsg, rootEventsCollection.Name(), err)
-		logger.WithError(err).Error("Could not drop root event collection.")
+		logger.Errorf("Could not drop root event collection: %v", err)
```

Comment:
same here

Reply:
I have moved the error logging to the top most level in each case now